### DISCUSSION TITLE
User Stories

### DISCUSSION THREAD

**Discussion by meganindya** [sugarlabs/musicblocks-v4#1] [Ideas]
https://app.leanboard.io/board/23e6d581-3120-49d4-8f55-ceed1a8b6190

![de2986c0-1688-42ec-b069-a48cfe728542](https://user-images.githubusercontent.com/33328728/101639412-4a69a200-3a55-11eb-9e99-b95140f6e78c.png)

Please use this _LeanBoard_ above to post all your perspectives about what little things this application should address. 

Here's an [article](https://www.atlassian.com/agile/project-management/epics-stories-themes) to get an idea about what we want to be doing. I'm guessing we can put ourselves in the shoes of an end user (multiple age groups), a teacher, musician, developer, etc.

Since we already have a working _Music Blocks_ application, we have a good idea about the big-picture and different issues associated with the current build.

**Comment by pikurasa** [comment]
@meganindya The LeanBoard link seems to have expired. Is there a way to easily regenerate it? If not, it's ok.
